src/test/kotlin/com/ampnet/reportservice/controller/AdminControllerTest.kt

```diff
@@ -65,7 +66,34 @@ class AdminControllerTest : ControllerTestBase() {
 
             val pdfContent = result.response.contentAsByteArray
             verifyPdfFormat(pdfContent)
-            // File(getDownloadDirectory(usersAccountsSummary)).writeBytes(pdfContent)
+            // File(getDownloadDirectory("usersAccountsSummary.pdf")).writeBytes(pdfContent)
+        }
+    }
+
+    @Test
+    @WithMockCrowdfundUser(privileges = [PrivilegeType.PRA_PROFILE])
+    fun mustBeAbleToDownloadXlsxForVerifiedUsers() {
```

Comment:
You could verify if it is in a valid xlsx format in the similar manner as we did with pdf with `verifyPdfFormat` method. Maybe apache poi lib has some useful method for that.